main.tf

```diff
@@ -150,10 +164,58 @@ resource "kubernetes_deployment" "wandb" {
             }
           }
         }
+
+        container {
+          name = "sidecar"
+          image = "busybox"
+
+          volume_mount {
+            name = "gorilla"
+            mount_path = "/var/log/${local.gorilla}"
+            sub_path = local.gorilla
+          }
+
+          volume_mount {
+            name = "local" 
+            mount_path = "/var/log/${local.local}"
+            sub_path = local.local
+          }
+
+          args = [ "/bin/sh", "-c", "tail -n+1 -f /var/log/gorilla.log" ]
```

Comment:
We're not doing anything with local.log in the sidecar?

Reply:
I was experimenting just getting this setup working with gorilla.log for now - planned on adding local.log later